llvm/lib/Transforms/Scalar/TailRecursionElimination.cpp

```diff
@@ -919,7 +964,8 @@ struct TailCallElim : public FunctionPass {
     return TailRecursionEliminator::eliminate(
         F, &getAnalysis<TargetTransformInfoWrapperPass>().getTTI(F),
         &getAnalysis<AAResultsWrapperPass>().getAAResults(),
-        &getAnalysis<OptimizationRemarkEmitterWrapperPass>().getORE(), DTU);
+        &getAnalysis<OptimizationRemarkEmitterWrapperPass>().getORE(), DTU,
+        nullptr);
```

Comment:
Document const param

Reply:
oh - because this is Legacy Pass Manager, and is code that should be deleted (see https://github.com/llvm/llvm-project/issues/82046). So no reason to add new features and support them there.

Reply:
ok, I'd still suggest documenting the const param though